### DISCUSSION TITLE
Scan interval on "total ac output power"

### DISCUSSION THREAD

**Discussion by Fjaslkhjvohas** [StephanJoubert/home_assistant_solarman#280] [Q&A]
I love this integration, and I've been testing my new solar system. However, I've been wondering why my status_lastUpdate refreshes every minute but my total_ac_output_power just updates every 5 minutes. 

Does someone know how to get updates of the power every minute as well?

My system:
Deye SUN600G3-EU-230
Used config file: deye_4mppt.yaml

Thanks for your help

**Comment by PV-Joe** [comment]
It’s the same with my Deye Hybrid. Not all values are updated every minute.

**Reply by lynntryanderror** [reply]
I opened the same topic today. Sorry for the double entry. 

Does the integration only revives the data or does it pulls the data from the inverter?
